.ci/functional_tests_utils.sh

```diff
@@ -76,6 +76,18 @@ function installDependencies() {
   installNodejs
 }
 
+function installDependenciesForCompatibilityCheck() {
+  installEpelRelease
+  installYQ
+  installJQ
+  installGit  
+  # Getting dependencies ready
+  yum install --assumeyes -d1 \
```

Comment:
Why do we need Java and Maven for the script if we comment `[test]` and exit?

Reply:
Because we are using that to change version here https://github.com/redhat-developer/rh-che/pull/1806/files#diff-946ba133a72dce6ae10a7d1b4f89575dR139

Reply:
would it be possible to do it from bash using `sed` or `yq`?

Reply:
Ok, let's keep Java and Maven there for now. Resolving discussion.